src/newSeed/stage4.ts

```diff
@@ -1,49 +1,61 @@
-      minDate: Date.now(),
-    });
-    // const dateService = new DateService();
-    startDatePicker.on("selected", (date: Date) => {
-      this.seedConfig.seedDetails.startDate.date = new Date(date.toDateString()).toLocaleDateString().replace(/\//g, "-");
-      // this.seedConfig.seedDetails.startDate.date = new Date(date.toDateString()).toUTCString();
-    });
-    const endDatePicker = new Litepicker({
-      element: this.endDate,
-      minDate: Date.now(),
-    });
-    endDatePicker.on("selected", (date: Date) => {
-      this.seedConfig.seedDetails.endDate.date = new Date(date.toDateString()).toLocaleDateString().replace(/\//g, "-");
-      // this.seedConfig.seedDetails.endDate.date = dateService.toISOString(date);
-    });
+    if (!this.startDatePicker) {
+      this.startDatePicker = new Litepicker({
+        element: this.startDateRef,
+        minDate: Date.now(),
+        autoRefresh: true,
+      });
+      this.startDatePicker.on("selected", (date: Date) => {
+        this.startDate = date;
+      });
+    }
+    if (!this.endDatePicker) {
+      this.endDatePicker = new Litepicker({
+        element: this.endDateRef,
+        minDate: Date.now(),
+        autoRefresh: true,
+      });
+      this.endDatePicker.on("selected", (date: Date) => {
+        this.endDate = date;
+      });
+    }
   }
   proceed(): void {
     let message: string;
-    if (!this.seedConfig.seedDetails.seedTokens || this.seedConfig.seedDetails.seedTokens.lte(0)) {
+    if (!this.seedConfig.seedDetails.seedTokens || this.seedConfig.seedDetails.seedTokens === "0") {
       message = "Please enter a value for the Amount of tokens to be added to SEED";
-    } else if (!this.seedConfig.seedDetails.pricePerToken || this.seedConfig.seedDetails.pricePerToken.lte(0)) {
+    } else if (!this.seedConfig.seedDetails.pricePerToken || this.seedConfig.seedDetails.pricePerToken === "0") {
       message = "Please enter a value for the Price per token";
-    } else if (!this.seedConfig.seedDetails.seedTarget || this.seedConfig.seedDetails.seedTarget.lte(0)) {
+    } else if (!this.seedConfig.seedDetails.seedTarget || this.seedConfig.seedDetails.seedTarget === "0") {
       message = "Please enter a non-zero value for the SEED Target";
-    } else if (!this.seedConfig.seedDetails.seedMax || this.seedConfig.seedDetails.seedMax.lte(0)) {
+    } else if (!this.seedConfig.seedDetails.seedMax || this.seedConfig.seedDetails.seedMax === "0") {
       message = "Please enter a non-zero number for the SEED Max";
     } else if (!this.seedConfig.seedDetails.vestingDays || this.seedConfig.seedDetails.vestingDays <= 0) {
       message = "Please enter a non-zero value for  \"Tokens vested for\" ";
     } else if (!this.seedConfig.seedDetails.vestingCliff || this.seedConfig.seedDetails.vestingCliff <= 0) {
       message = "Please enter a non-zero value for \"with a cliff of\" ";
-    } else if (!this.seedConfig.seedDetails.startDate) {
+    } else if (!this.startDate) {
```

Comment:
If it is possible to enter a malformed date, we should validate for it here.

Reply:
Regarding time validation: Can the user enter a negative number, and would it be accepted?

Reply:
No the user cannot enter a negative number